app/views/devices_guidance/preparing_4g_wireless_routers.md

```diff
@@ -47,11 +50,25 @@ When distributing devices, you should inform families, young people and care lea
 
 ## Router security
 
-Each router will only work with the SIM card that is provided with it. Attempting to use an alternative SIM in the router or use the SIM card with any other device will not give a data connection and may cause the SIM card to become barred.
+Each router will only work with the SIM card that is provided with it until it is reset. Attempting to use another SIM in the router before you reset it or use the SIM card with any other device will not give a data connection and may cause the SIM card to become barred.
```

Comment:
... it's reset ...

Attempting to use another SIM in the router before you reset it, or to use the SIM card with any other device will ...?